db/db_impl.cc

```diff
@@ -3054,5 +3061,68 @@ void DBImpl::WaitForIngestFile() {
   }
 }
 
+Status DBImpl::StartTrace(const TraceOptions& options,
+                          const std::string& trace_filename) {
+  EnvOptions env_options;
+  unique_ptr<WritableFile> trace_file;
```

Comment:
Using-directives is not allowed in Google C++ Style: https://google.github.io/styleguide/cppguide.html#Namespaces . I didn't check who added "using namespace std" to the code base but I hope we don't rely on it further.